index.html

```diff
@@ -111,6 +111,34 @@ <h2>Professional Software Development in 2018</h2>
           hodor - hodor hodor hodor. Hodor. Hodor hodor... Hodor hodor hodor hodor hodor? </p>
         <span class='expandButton'></span>
       </div>
+      <div class="article">
+        <h2>Cats Meow</h2>
+        <p class="date">Nov 7th, 2017</p>
+        <p>Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats  </p>
```

Comment:
I like the cats, but I wanna let you know about `lorem`. If you're using vscode you can type `lorem50` and hit tab to generate random text (50 is the number of words generated). Alternatively you could search for something like `business lorem` and find something more specific, but still gibberish